vertx/run-tests.zsh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env zsh
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+CHECK_CHAR='\U2713'
+CROSS_CHAR='\U2717'
+
+function print_result() {
+  version=$1
+  result=$2
+
+  if [[ $result == 0 ]];
+  then
+    mark=$CHECK_CHAR
+    color=$GREEN
+  else
+    mark=$CROSS_CHAR
+    color=$RED
+  fi
+
+  echo "\t${color}${version} ${mark}${NC}"
+}
+
+declare -A vertx_versions
+vertx_versions=( [v40x]="4.0.x", [v41x]="4.1.x", [v42x]="4.2.x", [v43x]="4.3.x", [v44x]="4.4.x", [v45x]="4.5.x" )
+v40x=( "4.0.2" )
+v41x=( "4.1.8" )
+v42x=( "4.2.7" )
+v43x=( "4.3.2" )
+v44x=( "4.4.9" )
+v45x=( "4.5.10" )
```

Comment:
so you test it with a matrix of versions?!

we need to mimic that into circle ci build

Reply:
I think, I even need to create separate submodule in feign-vertx to ensure a runtime compatibility with different versions of Vertx.

Do you have something similar in the project already ? Any suggestions how to integrate into CI ?